beacon_node/src/cli.rs

```diff
@@ -320,6 +320,16 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                 and never provide an untrusted URL.")
                 .takes_value(true),
         )
+        .arg(
+            Arg::with_name("monitoring-endpoint-frequency")
+                .long("monitoring-endpoint-frequency")
+                .value_name("SECONDS")
+                .help("Defines how many seconds to wait between each message sent to \
+                       the monitoring-endpoint.")
```

Comment:
How about:
```
.help(format("Defines how many seconds to wait between each message sent to the monitoring-endpoint. Default: {DEFAULT_UPDATE_DURATION}"))
```

Reply:
Unfortunately `clap` is a real pain in the ass and won't allow a `String` value to be inserted here:

```rust
error[E0515]: cannot return value referencing temporary value
   --> beacon_node/src/cli.rs:5:5
    |
5   |  /     App::new("beacon_node")
6   |  |         .visible_aliases(&["b", "bn", "beacon"])
7   |  |         .version(crate_version!())
8   |  |         .author("Sigma Prime <contact@sigmaprime.io>")
...    |
327 |  |                 .help(&format!("Defines how many seconds to wait between each message sent to \
    |  |________________________-
328 | ||                        the monitoring-endpoint. Default: {}s", 60))
    | ||__________________________________________________________________- temporary value created here
...    |
804 |  |                 .takes_value(false)
805 |  |         )
    |  |_________^ returns a value referencing data owned by the current function
```